src/data/matrix/basic.lean

```diff
@@ -1142,6 +1142,10 @@ lemma mul_vec_smul [fintype n] [monoid R] [non_unital_non_assoc_semiring S] [dis
   M.mul_vec (b • v) = b • M.mul_vec v :=
 by { ext i, simp only [mul_vec, dot_product, finset.smul_sum, pi.smul_apply, mul_smul_comm] }
 
+lemma vec_mul_single [fintype m] [decidable_eq m] (A : matrix m n α) (x : α) (i : m) :
+   vec_mul (pi.single i x) A = x • A i :=
+by { ext, simp [vec_mul, pi.single_apply] }
+
```

Comment:
Note these lemmas already appear in #13835

Reply:
Hm, yeah, should we wait for #13835 then?